Cargo.toml

```diff
@@ -143,6 +143,7 @@ evmap = { version = "7", features = ["bytes"] }
 logfmt = "0.0.2"
 notify = "4.0.14"
 once_cell = "1.3"
+getset = "0.1.0"
```

Comment:
When you get to import your own library! 

![](https://media.giphy.com/media/9PrvrbV4DiUUCiGPgQ/giphy.gif)